hpcgap/lib/init.g

```diff
@@ -548,16 +576,21 @@ BindGlobal( "ShowKernelInformation", function()
   if IsBound( GAPInfo.UseReadline ) then
     Add( libs, "readline" );
   fi;
+  if GAPInfo.KernelInfo.KernelDebug then
+    Add( libs, "KernelDebug" );
+  fi;
   if libs <> [] then
-    print_info( " Libs used:  ", libs, "\n" );
+    print_info( " Configuration:  ", libs, "\n" );
```

Comment:
Any test justifying this renaming?

Reply:
Justification: Syncing changes between GAP and HPC-GAP is literally what this PR is about... :-)

I.e. this was changed already quite some time ago in GAP, because we now list more than list libs. The variable `libs` should be renamed to something better eventually, but I deliberately did not, as this was about syncing the changes, not about cleaning `init.g` up.